prometheus-exporters/image-usage-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "image-usage-exporter.fullname" . }}
```

Comment:
There is no namespace set for the deployment. I would suggest putting it into `infra-monitoring` if there is no other namespace that fits better.